resources/splash.html

```diff
@@ -34,20 +25,72 @@
 	imagesdir: $imagesdir
 	pagesdir: $pagesdir
 
-	Variables that can be passed back via HTTP GET.
-	Just append them to the authentication link:
+Additional Variables that can also be passed back via HTTP get.
+Or just append them to the authentication link:
 	nodoguser
 	nodogpass
 	info
 	voucher
 -->
-	<a href="$authtarget">
-	<img src="$imagesdir/splash.jpg" width="71" height="49" border="0"
-		alt="Click to enter" title="Click to enter">
-	</a>
-	</td>
-</tr>
-</table>
+<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+<meta http-equiv="Pragma" content="no-cache" />
+<meta http-equiv="Expires" content="0" />
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel='shortcut icon' href='$imagesdir/splash.jpg' type='image/x-icon' />
+<title>$gatewayname Entry</title>
+<style>
+
+body
+{
+background-color:lightgrey; 
+color:black;
+margin-left: 5%;
+margin-right: 5%;
+text-align: left;
+}
+
+img
+{
+width: 40%;
+max-width: 180px;
+margin-left: 0%;
+margin-right: 5%;
+}
+
+input[type=submit]
+{
+color:black;
+margin-left: 0%;
+margin-right: 5%;
+text-align:left;   
+font-size: 1.0em;
+line-height: 2.5em;
+font-weight: bold;
+border: 1px solid;
+}
```

Comment:
I agree with some of your points.

ps - good, I already felt bad about being pedantic xD